tests/css_scss/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2025,7 +2025,7 @@ $my-map: (
 }
 
 // Ignore escape "\\" in SCSS mixins
-mixin margin-bottom-1\\/3 {
+@mixin margin-bottom-1\\/3 {
```

Comment:
Extra `\` (invalid `scss`), please test `scss` code here https://www.sassmeister.com/

Reply:
Oh sorry my mistake

Reply:
i've added
```scss
@mixin margin-bottom-1\/3 {
  margin-bottom: 0.8rem;
}

label {
  @include margin-bottom-1\/3;
}
```
to `tests/css_scss/scss.scss` and ran `yarn test -u`. This is auto generated